worker.js

```diff
@@ -63,40 +85,51 @@ var getBucketList = function (cb) {
 var readBuckets = function (tasks) {
   console.info('--- Started indexing all buckets ---');
   async.parallel(tasks,
```

Comment:
Probably not, at least for now. Only becomes a perf question when lots of buckets are being indexed.